lrauv_ignition_plugins/test/test_mission_pitch_mass.cc

```diff
@@ -89,19 +89,27 @@ TEST_F(LrauvTestFixture, PitchMass)
   // Euler.Z = yaw
   for(auto pose: this->tethysPoses)
   {
-    // Max Pitch 20 degrees. Allow 5 degrees error for oscillations.
-    // Min Pitch -5 degrees. Again allow 5 degrees error for oscillation.
+    // FIXME(hidmic): when surfacing, the vehicle oscillates
```

Comment:
I think the problem here is we are testing at the surface which makes behaviors hard to define and will into oscillations. Rather than tweaking this parameter, we should not derive from `LrauvTestFixture` but have the test derive from [`LrauvTestFixtureAtDepth`](https://github.com/osrf/lrauv/blob/b84780f0cb3848b84f0e454fd0fdf2dc2b30241e/lrauv_ignition_plugins/test/helper/LrauvTestFixture.hh#L320). I thought I had done this but its not clearly the case.

Reply:
On a second inspection, we still get an initial oscillation about pitch = 0 if running the test at a depth (which requires it to be neutrally buoyant) as soon as the LRAUV controller is engaged. I'll check for spurious mass shifter commands.